nautobot/extras/templates/extras/inc/jobresult.html

```diff
@@ -12,45 +12,43 @@
         <strong>Summary of Results</strong>
     </div>
     <table class="table table-hover panel-body">
-        {% if result.job_model is not None %}
             <tr>
                 <td>Job Description</td>
-                <td>{{ result.job_model.description | render_markdown }}</td>
+                <td>{{ result.job_model.description | render_markdown | placeholder }}</td>
             </tr>
-        {% endif %}
         <tr>
             <td>Status</td>
             <td><span id="pending-result-label">{% include 'extras/inc/job_label.html' with result=result %}</span></td>
         </tr>
         <tr>
             <td>Started at</td>
-            <td>{{ result.date_created }}</td>
+            <td>{{ result.date_created | placeholder }}</td>
         </tr>
         <tr>
             <td>User</td>
-            <td>{{ result.user }}</td>
+            <td>{{ result.user | placeholder }}</td>
         </tr>
         <tr>
             <td>Duration</td>
             <td>
-            {% if result.date_done %}
-                {{ result.duration }}
-            {% else %}
+            {% if result.date_created and not result.date_done %}
                 <img src="{% static 'img/ajax-loader.gif' %}">
+            {% else %}
+                {{ result.duration | placeholder}}
             {% endif %}
             </td>
         </tr>
         <tr>
             <td>Return Value</td>
             <td>
-            {% if result.date_done %}
+            {% if result.date_created and not result.date_done %}
+                <img src="{% static 'img/ajax-loader.gif' %}">
+            {% else %}
                 {% if result.result %}
                     <pre>{{ result.result | render_json }}</pre>
                 {% else %}
                     {{ result.result | placeholder }}
                 {% endif %}
```

Comment:
This conditional was here before my changes. In my testing, it seems unnecessary, but I have left it in case there's some scenario I'm not expecting.

Reply:
Looking at it I **think** it's just so that the placeholder doesn't get unnecessarily wrapped in a `<pre>` tag, but that may not be an issue in practice.

Reply:
Yeah my experience when I tested it was that it was not an issue when it did. But I'll leave it nonetheless.